styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
+        border-radius: 16px;
+        padding: 3rem 4rem;
+        display: flex;
+        flex-direction: column;
+        // Color /
+        background: #FFFFFF;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+        // Text /
+        text-align: left;
+    }
+    &__heading {
+        margin: 0 auto;
+        text-align: center;
+        text-transform: uppercase;
+        line-height: 51px;
+    }
+    &__text {
+        margin: 1rem 0;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 34px;
+    }
+}
+
+.details {
+    margin-left: 285px;
+    height: 355px;
+    padding-top: .75rem;
+    &__heading {
+        margin-bottom: .2rem;
+        margin-left: 50px;
+        font-weight: 600;
```

Comment:
make sure this fw is being used across the project. I guess you could cleanup unused font imports if any

Reply:
Figma specifies fw for all the text, so they vary depending on where they are used.

Reply:
I guess it also specifies font styles (not CSS property, but a set of properties). Check it please and let me know if any

Reply:
Yes, it does, what I meant is that they vary, so it had to be checked every time.

Reply:
Usually components in figma have text styles and color styles, consider it as a future design tip 😉

Reply:
Yes, but the other properties are defined elsewhere, so they were not necessary here.